vrs/vrs_annotation_batch.py

```diff
@@ -119,157 +128,177 @@ def main(args):
+            vrs_script_path = (
+                "/usr/local/lib/python3.9/dist-packages/ga4gh/vrs/extras/vcf_annotation.py"
+            )
 
-    logger.info(
-        f"Gathered list of files in gs://{working_bucket}/vrs-temp/shard-{version}.vcf.bgz using Hail's Hadoop method"
-    )
+            # Store VCF input and output paths
+            vcf_input = f'gs://{working_bucket}/vrs-temp/shard-{version}.vcf.bgz/{vcf_name}'
+            vcf_output = f'/temp-vcf-annotated/annotated-{vcf_name.split(".")[0]}.vcf'
 
-    # Create a list of all shards of VCF
-    file_dict = hl.utils.hadoop_ls(
-        f"gs://{working_bucket}/vrs-temp/shard-{version}.vcf.bgz/part-*.bgz"
-    )
+            # Perform VRS annotation on vcf_input and store output in /temp-vcf-annotated
+            new_job.command("mkdir /temp-vcf-annotated/")
+            new_job.command(
+                f"python3 {vrs_script_path} --vcf_in {batch_vrs.read_input(vcf_input)} --vcf_out {vcf_output} --seqrepo_root_dir {seqrepo_path} --vrs_attributes"
+            )
 
-    # Create a list of all file names to later annotate in parallel
-    file_list = [file_item["path"].split("/")[-1] for file_item in file_dict]
+            # Copy annotated shard to its appropriate place in Google Bucket
+            new_job.command(
+                f"gsutil cp {vcf_output} gs://{working_bucket}/vrs-temp/annotated-{version}.vcf/"
+            )
 
-    # Query-On-Batch produces duplicate shards (same number, same content, different name) for v3.1.2 Release Table
-    # This block removes duplicates from the list of files to be annotated
-    to_exclude = []
-    for file_idx in range(len(file_list)): 
-        file_name = file_list[file_idx].split("-")
-        file_num = file_name[1]
-        if file_list[file_idx-1].split("-")[1] == file_num:
-            to_exclude.append(file_list[file_idx-1])
+        # Execute all jobs in Batch
+        batch_vrs.run()
 
-    file_list = sorted(list(set(file_list) - set(to_exclude)))
-    logger.info(f"Number of duplicates to be excluded: {len(to_exclude)}")
+        logger.info("Batch Jobs executed, preparing to read in sharded VCF from prior step")
 
-    logger.info("File list created... getting ready to start Batch Jobs")
+        # Import all annotated shards
+        ht_annotated = hl.import_vcf(
+            f"gs://{working_bucket}/vrs-temp/annotated-{version}.vcf/*.vcf",
+            reference_genome="GRCh38",
+        ).make_table()
+        logger.info("Annotated table constructed")
 
-    # Create backend and batch for coming annotation batch jobs
-    backend = hb.ServiceBackend(args.billing_project, working_bucket)
+        # When you select something, even in the 'info' field, it is no longer in that struct!
+        ht_annotated = ht_annotated.select(ht_annotated.info.VRS_Allele,ht_annotated.info.VRS_Start,ht_annotated.info.VRS_End,ht_annotated.info.VRS_Alt)
```

Comment:
can you check in with Mike so that he knows how we are planning on adding these annotations (and also confirm that using their default capitalization is fine)?